scalikejdbc-core/src/main/scala/scalikejdbc/ScalaBigIntegerConverterImplicits.scala

```diff
@@ -0,0 +1,14 @@
+package scalikejdbc
+
+import scala.language.implicitConversions
+
+/**
+ * Implicit conversions for BigInteger values.
+ */
+trait ScalaBigIntegerConverterImplicits {
```

Comment:
I think it's ok to enable this trait by default as same as BigDecimal's one. Or is there any reason we should avoid to do so? https://github.com/scalikejdbc/scalikejdbc/blob/2.3.4/scalikejdbc-library/src/main/scala/scalikejdbc/package.scala#L52